synapse/lib/stormlib/infosec.py

```diff
@@ -858,32 +709,33 @@ async def calculateFromProps(self, props, vers='3.1'):
     async def vectToScore(self, vect, vers=None):
         vers = await s_stormtypes.tostr(vers, noneok=True)
 
-        if vers not in CVSS_VERSIONS + [None]:
-            raise s_exc.BadArg(mesg=f'Valid values for vers are: {CVSS_VERSIONS + [None]}.')
+        if vers not in s_cvss.versions + [None]:
+            raise s_exc.BadArg(mesg=f'Valid values for vers are: {s_cvss.versions + [None]}, got {vers}')
 
-        detected = CVSS3_1_STR
+        detected = s_cvss.cvss3_1
 
         if vers is None:
             if 'Au:' in vect or vect.startswith('CVSS#2'):
-                detected = CVSS2_STR
+                detected = s_cvss.cvss2
 
             elif vect.startswith('CVSS:3.0/'):
-                detected = CVSS3_0_STR
+                detected = s_cvss.cvss3_0
 
             elif vect.startswith('CVSS:3.1/'):
-                detected = CVSS3_1_STR
+                detected = s_cvss.cvss3_1
 
         else:
             detected = vers
 
-        vdict = validate(vect, detected)
+        vdict = s_chop.cvss_validate(vect, detected)
 
         calc = CVSS_CALC[detected]
         base, temporal, environmental, overall = calc(vdict)
 
         return {
             'version': detected,
             'score': overall,
+            'normalized': s_chop.cvss_normalize(vdict, detected),
```

Comment:
Do you want to add this to the return value docstring?

Reply:
Yep, thanks.